app/webpack.common.ts

```diff
@@ -81,7 +81,7 @@ export const renderer = merge({}, commonConfig, {
       },
       {
         test: /\.cmd$/,
-        loader: 'file-loader',
```

Comment:
With update to node came a deprecation of of using the open ssl in the `file-loader` plugin. Markus found that Webpack version we are on no longer needs `file-loader`. Thus we removed this dependency as it was not working with newer node version.